en/docs/learn/the-evolution-of-provisioning-standards.md

```diff
@@ -77,7 +77,7 @@ played a major role in building the SPML 1.0.
 1.  [Information Technology Markup Language
     (ITML)](http://xml.coverpages.org/itml.html)
 2.  [Active Digital Profile (ADPr)](http://xml.coverpages.org/adpr.html)
-3.  [eXtensible Resource Provisioning Management
```

Comment:
eXtensible Resource Provisioning Management is how it is usually referred to in other docs probably owing to its abbreviation (XRPM)

Reply:
Closing the PR for the above issue based on the above comment